catalog/ui/search-ui/standard/src/main/webapp/js/view/preferences/PreferencesModal.view.js

```diff
@@ -320,9 +320,10 @@ define([
             ui: {
                 tbody: 'tbody'
             },
+            viewComparator: 'label',
             initialize: function() {
                 this.collection.each(function(model) {
```

Comment:
The latest Backbone (1.30-1.33) has a bug with this behavior, so I've opened an issue here: https://github.com/jashkenas/backbone/issues/4064